src/platform/tls_schannel.c

```diff
@@ -3058,7 +3058,7 @@ CxPlatTlsParamGet(
             break;
 
         case QUIC_PARAM_TLS_HANDSHAKE_INFO: {
-            if (*BufferLength < sizeof(QUIC_HANDSHAKE_INFO)) {
+            if (*BufferLength < CXPLAT_STRUCT_SIZE_THRU_FIELD(QUIC_HANDSHAKE_INFO, CipherSuite)) {
```

Comment:
Won't this need to be updated every time the `QUIC_HANDSHAKE_INFO` is changed?
Why is sizeof insufficient? To prevent failure of older applications using a newer MsQuic.dll passing the old struct size?

Reply:
Ah yeah, that's exactly it. and no it doesn't need to be updated. I see it now.